news/services/embedding.py

```
@@ -0,0 +1,46 @@
+                    model=self.model,
+                    content=text,
+                    task_type="retrieval_document",
+                )
+                # 응답 형식: {"embedding": [0.1, 0.2, ...]}
+                if "embedding" in result:
+                    embeddings.append(result["embedding"])
+                else:
+                    logger.warning(f"Unexpected embedding response format: {result}")
+                    embeddings.append(None)
+
+            return embeddings
```

Ruff (0.14.10)

Consider moving this statement to an `else` block

(TRY300)